resources/views/deletefaq.blade.php

```diff
@@ -0,0 +1,25 @@
+<?php
+use Illuminate\Support\Facades\DB
+?>
+
+<x-layout>
+    <x-slot:title>
+        Delete FAQ Post {{ $post }}
+    </x-slot>
+    <form method="post" action="/deletefaqpost">
+        @method('put')
+        @csrf
+        <h1>Vraag: </h1>
+        <input type="text" name="question" value="{{ DB::table('faq')->where('id', $post)->value('question') }}">
```

Comment:
`DB::table(...)` works, but it is better to use models instead. It will make your code more maintainable and easier to extend, with business logic for example.